### PR TITLE
Wip/bewest/usb init

### PR DESCRIPTION
**Pull request by bewest**

### CODE REVIEW

File: tools/serial_rf_spy.py [hunk 1]
```diff
@@ -16,7 +16,8 @@ def __init__(self, serial_port, rtscts=None):
     if not rtscts:
       rtscts = int(os.getenv('RFSPY_RTSCTS', 1))
 
-    self.ser = serial.Serial(serial_port, 19200, rtscts=rtscts, timeout=1)
+    # self.ser = serial.Serial(serial_port, 19200, rtscts=rtscts, timeout=1)
+    self.ser = serial.Serial(serial_port, 115200, timeout=1)
```
Comment (ps2) [inline tools/serial_rf_spy.py]: Is this intentional (19200 -> 115200) ?  I think it will not work with other serial implementations.

Comment (bewest) [inline tools/serial_rf_spy.py]: Whole commit should be flagged... this is quick debugging, not intentional.

**Comment by ps2** [comment]
Is it possible to move the target branch of a PR?  I'd like to pull this into dev.

**Comment by ps2** [comment]
Merged into dev.

**Pull request closed without merging**